src/utils/provideLocalStorageDataClass.ts

```diff
@@ -91,3 +91,33 @@ function isStringObject(item: unknown): item is Record<string, unknown> {
   if (!isObject(item)) return false
   return Object.keys(item).every((key) => typeof key === 'string')
 }
+
+function sortItems(
+  items: [string, unknown][],
+  order: Array<[string, 'asc' | 'desc']>,
+): [string, unknown][] {
+  if (order.length === 0) return items
+
+  return [...items].sort((a, b) => {
+    const itemA = a[1]
+    const itemB = b[1]
+    if (!isObject(itemA) || !isObject(itemB)) return 0
+
+    for (const [attr, ascOrDesc] of order) {
+      const valueA = (itemA as Record<string, unknown>)[attr]
```

Comment:
* [ ] Suspicious cast. `object` suddenly becomes a more special type. We should use a better type guard than `isObject` and get rid of the cast. `isStringObject` is close, we probably need something like `isStringNumberObject`.